h/feeds/atom.py

```diff
@@ -56,6 +58,8 @@ def feed_from_annotations(
     :rtype: dict
 
     """
+    annotations = [h.AnnotationHTMLPresenter(a) for a in annotations]
```

Comment:
It doesn't make sense to be using something called `AnnotationHTMLPresenter` in a module that isn't rendering to HTML. I"m also not sure at all about having to then dereference the original annotation with `annotation.annotation` earlier.

What's the need to use the presenter at all in this module?